src/core/LoadBalancer.ts

```diff
@@ -59,7 +59,7 @@ export interface LoadBalancer {
     /**
      * Returns whether this instance supports getting data members through a call to {@link nextDataMember()}.
      *
-     * @return Returns the next member or `null` if no member is available
+     * @return Returns if this load balancer can get a data member.
```

Comment:
`@return Returns if this load balancer can get a data member.` -> `@return `True` if this load balancer can get a data member.`

Reply:
I think old version is still ok, but I will change. I used `if` in the same meaning with `whether`